### PR TITLE
Add Nix development environment

### PR DESCRIPTION
**Pull request by link2xt**
It is easier to setup and use than the Dockerfile.

### CODE REVIEW

File: flake.lock [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        "lastModified": 1677407201,
+        "narHash": "sha256-3blwdI9o1BAprkvlByHvtEm5HAIRn/XPjtcfiunpY7s=",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "7f5639fa3b68054ca0b062866dc62b22c3f11505",
+        "type": "github"
+      },
+      "original": {
+        "owner": "NixOS",
+        "ref": "nixos-unstable",
```
Comment (dotlambda) [inline flake.lock:120]: I suggest
```suggestion
        "ref": "nixpkgs-unstable",
```
for stuff that's not related to NixOS.

File: flake.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  description = "Delta Chat for Android";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```
Comment (dotlambda) [inline flake.nix:5]: I suggest
```suggestion
    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
```
for stuff that's not related to NixOS.

**Review by r10s — APPROVED** [review]
lgtm - i am not using Docker nor Nix, so i mainly reviewed that things will work on "standard" dev environment

**Comment by r10s** [comment]
> @r10s Could you check that ndk-make.sh with fixes still works for you?

@link2xt i just tried ndk-make.sh with this pr on my machine - completed successfully on my machine :)

**Pull request closed without merging**